### PR TITLE
PMM-7087 Rename custom query example file

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-7087

Ref: https://github.com/percona/postgres_exporter/pull/43

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -29,7 +29,7 @@
 [submodule "postgres_exporter"]
 	path = sources/postgres_exporter/src/github.com/percona/postgres_exporter
 	url = https://github.com/percona/postgres_exporter
-	branch = master
+	branch = PMM-7087-rename-custom-query-example-file
```
Comment (askomorokhov) [inline .gitmodules]: Change back to master after testing before merge.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1880-66fa98e
client docker - perconalab/pmm-client-fb:PR-1880-66fa98e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1880-66fa98e.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1880-66fa98e&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1880-66fa98e.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/9015/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/9015/

**Comment by puneet0191** [comment]
UI tests failure looks like false positive on this one will fix it separately.

**Review by percona-csalguero — DISMISSED** [review]

**Review by nikita-b — DISMISSED** [review]

**Comment by nikita-b** [comment]
LGTM

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1880-e1202f3
client docker - perconalab/pmm-client-fb:PR-1880-e1202f3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1880-e1202f3.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1880-e1202f3&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1880-e1202f3.tar.gz

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/6786/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/9085/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1880-5367918
client docker - perconalab/pmm-client-fb:PR-1880-5367918
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1880-5367918.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1880-5367918&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1880-5367918.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1880-3672c34
client docker - perconalab/pmm-client-fb:PR-1880-3672c34
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1880-3672c34.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1880-3672c34&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1880-3672c34.tar.gz

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/7005/

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/7006/

**Pull request merged**